autotst/reaction.py

```diff
@@ -687,13 +687,22 @@ def get_rdkit_mol(self):
             rd_copy = rdkit.Chem.RWMol(self.rdkit_molecule.__copy__())
 
             lbl1, lbl2, lbl3 = self.labels
-
-            if not rd_copy.GetBondBetweenAtoms(lbl1, lbl2):
-                rd_copy.AddBond(lbl1, lbl2,
-                                order=rdkit.Chem.rdchem.BondType.SINGLE)
-            elif not rd_copy.GetBondBetweenAtoms(lbl2, lbl3):
-                rd_copy.AddBond(lbl2, lbl3,
-                                order=rdkit.Chem.rdchem.BondType.SINGLE)
+            if self.reaction_family.lower() in ['h_abstraction', 'r_addition_multiplebond', 'disproportionation']:
```

Comment:
Instead of hard coding, perhaps use reaction family recipe to determine where to make the fake bonds